packages/moonstone/Panels/Header.js

```diff
@@ -246,7 +246,7 @@ const HeaderBase = kind({
 		delete rest.subTitleBelow;
 		delete rest.titleBelow;
 
-		const titleHeight = ri.unit(ri.scale((type === 'dense') ? 69 : 90), 'rem');
+		const titleHeight = (type === 'dense') ? 69 : 90;
```

Comment:
Would it be worth documenting this as almost all other measurements _aren't_ in rem?

Reply:
It was documented in `ui/Layout.Cell#size`, but I can't find that documentation now. `size` supports strings, which are applied literally to the size rule, or as a number, which is converted safely to `rem` inside `Cell`. There are even examples of this in the docs, so I'm not sure where the docs went.